src/FileSystem-Tests-Core/FileSystemTest.class.st

```diff
@@ -447,7 +447,7 @@ FileSystemTest >> testFile [
 	path := Path * 'gooly'.
 	self markForCleanup: path.
 	
-	(filesystem open: path writable: true) writeStream close.
+	(filesystem workingDirectory resolve: path) writeStream close.
```

Comment:
It would be nice to avoid #workingDirectory usage for tests because actual folder can not be writable.
For example if I will run image using Finder on Mac these tests will fail because directory will be root (/).

We should use FileLocator temp if possible

Reply:
Yeh, but I just rewrote the tests to a version equivalent to what they did before.
I did not refactor them...

Probably we should do that in another issue.

Reply:
Agree. I opened https://pharo.fogbugz.com/f/cases/21625/We-should-avoid-workingDirectory-to-be-used-for-temp-test-files